### ISSUE TITLE
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://liliputing.com/feed"))

### ISSUE THREAD

**Issue by gregf** [skeeto/elfeed#373]
I'm seeing some errors when updating my feeds. I'm not sure if this is a elfeed-org issue (maybe something I have done wrong) or something with elfeed itself.

I see a bunch of these messages when updating my feeds though.

```
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://liliputing.com/feed"))
Quit
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://about.gitlab.com/atom.xml"))
Quit
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://about.gitlab.com/atom.xml"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("http://planet.emacsen.org/atom.xml"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://about.gitlab.com/atom.xml"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("http://www.terminally-incoherent.com/blog/feed"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://blog.netbsd.org/tnf/feed/entries/atom" "https://blog.netbsd.org/tnf/feed/entries/atom" "https://blog.netbsd.org/tnf/feed/entries/atom"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://about.gitlab.com/atom.xml" "https://about.gitlab.com/atom.xml"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp (("http://nullprogram.com/feed") ("https://hothardware.com/rss/news.aspx") ("https://news.google.com/news/rss") ("https://reddit.com/r/linux_admin/.rss" "https://reddit.com/r/linux_gaming/.rss" "https://reddit.com/r/emacs/.rss" "https://reddit.com/r/openbsd/.rss" "https://reddit.com/r/freebsd/.rss" "https://reddit.com/r/bsd/.rss") ("https://www.gamingonlinux.com/article_rss.php") ("http://feeds.feedburner.com/metalinjection") ("https://lwn.net/headlines/newrss") ("https://lobste.rs/rss?token=3Nu7TkSJUGJxA8e6fg813FkKbLXSAIPCse8NJSCAzLEQxcQ6UATouSwu4Fbu") ("https://xkcd.com/rss.xml") ("https://github.com/gregf.private.atom?token=AAAD1NnRmIcN7wG2qYJUMN27WiCeCoRiks66feGRwA==") ("https://tilde.news/rss") ("https://liliputing.com/feed") ("http://blog.humblebundle.com/rss") ("https://www.osnews.com/feed/") ("http://planet.emacsen.org/atom.xml") ("https://about.gitlab.com/atom.xml") ("http://feeds.arstechnica.com/arstechnica/index/") ("http://www.terminally-incoherent.com/blog/feed") ("https://grafana.com/blog/index.xml") ("https://feeds.feedburner.com/Torrentfreak") ("https://www.blabbermouth.net/feed.rss") ("https://www.phoronix.com/rss.php") ("https://sachachua.com/blog/category/emacs/feed/") ("http://www.freshports.org/backend/rss2.0.php")))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("http://blog.humblebundle.com/rss"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp (("https://www.reddit.com/r/devops/.rss" "https://www.reddit.com/r/ansible/.rss" "https://www.reddit.com/r/TronScript/.rss" "https://www.reddit.com/r/zsh/.rss" "https://www.reddit.com/r/unix/.rss" "https://www.reddit.com/r/steamdeals/.rss" "https://www.reddit.com/r/technology/.rss" "https://www.reddit.com/r/television/.rss" "https://www.reddit.com/r/commandline/.rss" "https://www.reddit.com/r/debian/.rss" "https://www.reddit.com/r/movies/.rss" "https://www.reddit.com/r/golang/.rss" "https://www.reddit.com/r/ansible/.rss" "https://www.reddit.com/r/TronScript/.rss" "https://www.reddit.com/r/unix/.rss" "https://www.reddit.com/r/technology/.rss" "https://www.reddit.com/r/television/.rss" "https://www.reddit.com/r/commandline/.rss" "https://www.reddit.com/r/golang/.rss" "https://www.reddit.com/r/zsh/.rss" "https://www.reddit.com/r/debian/.rss" "https://www.reddit.com/r/steamdeals/.rss" "https://www.reddit.com/r/devops/.rss" "https://www.reddit.com/r/movies/.rss") ("https://hothardware.com/rss/news.aspx" "https://hothardware.com/rss/news.aspx") ("https://reddit.com/r/emacs/.rss" "https://reddit.com/r/openbsd/.rss" "https://reddit.com/r/linux_gaming/.rss" "https://reddit.com/r/bsd/.rss" "https://reddit.com/r/linux_admin/.rss" "https://reddit.com/r/freebsd/.rss" "https://reddit.com/r/emacs/.rss" "https://reddit.com/r/freebsd/.rss" "https://reddit.com/r/bsd/.rss" "https://reddit.com/r/linux_admin/.rss" "https://reddit.com/r/openbsd/.rss" "https://reddit.com/r/linux_gaming/.rss") ("http://feeds.feedburner.com/metalinjection" "http://feeds.feedburner.com/metalinjection") ("http://nullprogram.com/feed" "http://nullprogram.com/feed") ("https://www.gamingonlinux.com/article_rss.php" "https://www.gamingonlinux.com/article_rss.php") ("https://www.osnews.com/feed/" "https://www.osnews.com/feed/") ("https://github.com/gregf.private.atom?token=AAAD1NnRmIcN7wG2qYJUMN27WiCeCoRiks66feGRwA==" "https://github.com/gregf.private.atom?token=AAAD1NnRmIcN7wG2qYJUMN27WiCeCoRiks66feGRwA==") ("http://feeds.arstechnica.com/arstechnica/index/" "http://feeds.arstechnica.com/arstechnica/index/") ("https://www.phoronix.com/rss.php" "https://www.phoronix.com/rss.php") ("http://www.terminally-incoherent.com/blog/feed" "http://www.terminally-incoherent.com/blog/feed") ("https://blog.digitalocean.com/rss/" "https://blog.digitalocean.com/rss/") ("https://xkcd.com/rss.xml" "https://xkcd.com/rss.xml") ("http://www.freshports.org/backend/rss2.0.php" "http://www.freshports.org/backend/rss2.0.php") ("http://planet.emacsen.org/atom.xml" "http://planet.emacsen.org/atom.xml") ("https://lobste.rs/rss?token=3Nu7TkSJUGJxA8e6fg813FkKbLXSAIPCse8NJSCAzLEQxcQ6UATouSwu4Fbu" "https://lobste.rs/rss?token=3Nu7TkSJUGJxA8e6fg813FkKbLXSAIPCse8NJSCAzLEQxcQ6UATouSwu4Fbu") ("https://sachachua.com/blog/category/emacs/feed/" "https://sachachua.com/blog/category/emacs/feed/") ("https://lwn.net/headlines/newrss" "https://lwn.net/headlines/newrss") ("https://blog.netbsd.org/tnf/feed/entries/atom" "https://blog.netbsd.org/tnf/feed/entries/atom") ("https://tilde.news/rss" "https://tilde.news/rss") ("https://news.google.com/news/rss" "https://news.google.com/news/rss") ("http://blog.humblebundle.com/rss" "http://blog.humblebundle.com/rss") ("https://about.gitlab.com/atom.xml" "https://about.gitlab.com/atom.xml") ("https://feeds.feedburner.com/Torrentfreak" "https://feeds.feedburner.com/Torrentfreak")))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://grafana.com/blog/index.xml"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://www.reddit.com/r/golang/.rss" "https://www.reddit.com/r/devops/.rss" "https://www.reddit.com/r/technology/.rss" "https://www.reddit.com/r/zsh/.rss" "https://www.reddit.com/r/movies/.rss" "https://www.reddit.com/r/steamdeals/.rss" "https://www.reddit.com/r/debian/.rss" "https://www.reddit.com/r/TronScript/.rss" "https://www.reddit.com/r/unix/.rss" "https://www.reddit.com/r/ansible/.rss" "https://www.reddit.com/r/commandline/.rss" "https://www.reddit.com/r/television/.rss"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://grafana.com/blog/index.xml"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://www.reddit.com/r/golang/.rss" "https://www.reddit.com/r/zsh/.rss" "https://www.reddit.com/r/ansible/.rss" "https://www.reddit.com/r/steamdeals/.rss" "https://www.reddit.com/r/TronScript/.rss" "https://www.reddit.com/r/movies/.rss" "https://www.reddit.com/r/unix/.rss" "https://www.reddit.com/r/television/.rss" "https://www.reddit.com/r/devops/.rss" "https://www.reddit.com/r/commandline/.rss" "https://www.reddit.com/r/technology/.rss" "https://www.reddit.com/r/debian/.rss" "https://www.reddit.com/r/ansible/.rss" "https://www.reddit.com/r/zsh/.rss" "https://www.reddit.com/r/technology/.rss" "https://www.reddit.com/r/golang/.rss" "https://www.reddit.com/r/steamdeals/.rss" "https://www.reddit.com/r/unix/.rss" "https://www.reddit.com/r/debian/.rss" "https://www.reddit.com/r/devops/.rss" "https://www.reddit.com/r/television/.rss" "https://www.reddit.com/r/TronScript/.rss" "https://www.reddit.com/r/commandline/.rss" "https://www.reddit.com/r/movies/.rss" "https://www.reddit.com/r/devops/.rss" "https://www.reddit.com/r/debian/.rss" "https://www.reddit.com/r/unix/.rss" "https://www.reddit.com/r/commandline/.rss" "https://www.reddit.com/r/steamdeals/.rss" "https://www.reddit.com/r/golang/.rss" "https://www.reddit.com/r/TronScript/.rss" "https://www.reddit.com/r/television/.rss" "https://www.reddit.com/r/technology/.rss" "https://www.reddit.com/r/ansible/.rss" "https://www.reddit.com/r/movies/.rss" "https://www.reddit.com/r/zsh/.rss"))
Error running timer ‘elfeed-curl--call-callback’: (wrong-type-argument stringp ("https://www.reddit.com/r/TronScript/.rss" "https://www.reddit.com/r/steamdeals/.rss" "https://www.reddit.com/r/zsh/.rss" "https://www.reddit.com/r/television/.rss" "https://www.reddit.com/r/ansible/.rss" "https://www.reddit.com/r/commandline/.rss" "https://www.reddit.com/r/debian/.rss" "https://www.reddit.com/r/golang/.rss" "https://www.reddit.com/r/technology/.rss" "https://www.reddit.com/r/unix/.rss" "https://www.reddit.com/r/movies/.rss" "https://www.reddit.com/r/devops/.rss"))
```

I'm using elfeed  20200209.1942 from melpa with doom emacs.

**Comment by gregf** [comment]
There was a issue in my elfeed.org file causing this i believe. Either way its sorted now.

**Comment by Blade6570** [comment]
What was the issue?

**Comment by gregf** [comment]
Sorry I don't really remember anymore other than something was formatted wrong in my list file.

**Comment by minad** [comment]
Fixed by #555, commit bbb3cac27b0412d80b327b5cfaab83683c96a2a1
